src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -569,8 +569,8 @@ class CollectDuplexSeqMetrics
     */
   private def takeNextGroup(iterator: BetterBufferedIterator[SamRecord]): Seq[SamRecord] = {
     val rec = iterator.head
-    val info = GroupReadsByUmi.ReadInfo(rec)
-    iterator.takeWhile(rec => GroupReadsByUmi.ReadInfo(rec) == info).toIndexedSeq
+    val info = GroupReadsByUmi.ReadInfo(rec, ignoreStrand=rec.unpaired)
+    iterator.takeWhile(rec => GroupReadsByUmi.ReadInfo(rec, ignoreStrand=rec.unpaired) == info).toIndexedSeq
```

Comment:
Won't this now do the wrong thing for single UMI data?  I.e. if you have two reads who happen to have the same unclipped 5' end, but are on opposite strands, they'll now show up as having the same ReadInfo, and get bundled together for grouping?